CHANGELOG.md

```diff
@@ -14,12 +14,14 @@ run time expected as a result.
 - Warning if PML or absorbing boundaries are used along a simulation dimension with zero size.
 
 ### Changed
+- Issue a deprecation warning that `Geometry.intersections` will be renamed to `Geometry.intersections_plane` in 2.0.
 
 
 ### Fixed
 - Progressbars always set to 100% when webapi functions are finished.
 - Faster handling of `Geometry.intersects` and `Geometry.inside` by taking into account geometry bounds.
 - Numpy divide by zero warning in mode solver fixed by initializing jacobians as real instead of complex.
+- Bug in validators for 2D objects being in homogeneous media which were looking at the infinite plane in which the objects lie.
```

Comment:
feel like you can make this a bit stronger, to mention that there's a big speedup for many structures?